components/bookshelf/finished/FinishedList.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect, Dispatch, SetStateAction } from "react"
+import Book from "../Book"
+import { BookType } from "api/types"
+import { getFinishedList } from "api/books/finishedlist"
+import Header from "../Header"
+
+interface ReadingListProp {
+  setActive: Dispatch<SetStateAction<string>>
+}
+
+const FinishedList = ({ setActive }: ReadingListProp) => {
+  const [books, setBooks] = useState<BookType[]>([])
+
+  useEffect(() => {
+    const fetchBooks = async () => {
+      const bookList = await getFinishedList()
```

Comment:
Does `bookList` automatically infer the `BookList` type?

Reply:
tbh, I'm not sure. I checked it kanina, before making the changes in the comment below,  and it showed 'booklist: any'. so it probably didn't. it also didn't give the red error/underline so I didn't check beforehand thinking its all good. my bad on this one.

Reply:
> booklist: any

Hmmm can you try running `npm run build` locally? Does it build?